report/3.Theory/1.TrackingObjects.tex

```diff
@@ -0,0 +1,27 @@
+\section{Predicting a moving objects location}
+In order to predict a moving objects location a keen observation to be made is whether the object is flying i.e{.} if the object is following a ballistic trajectory or whether the object is driving i.e{.} following a linear trajectory. 
```

Comment:
`in order to predict the location of a moving object`.
`to be made*,* is wheather`